src/components/Header.tsx

```diff
@@ -21,12 +21,24 @@ export default function Header() {
 
     useEffect(() => {
```

Comment:
use `useMount` from `@/hooks/use-mount` instead of `useEffect`

another better suggestion is [A third party event listener hook](https://usehooks-ts.com/react-hook/use-event-listener) which optimise event listener in react. (It's ok for only use `useMount` if you don't want to install a third party hook lib)

Reply:
I'll just use useMount for now.